app/views/passengers/_form.html.haml

```diff
@@ -42,8 +42,8 @@
      options_from_collection_for_select(MobilityDevice.order(:name),
       :id, :name), include_blank: 'None'
   .field
-    = form.label :active
-    = form.check_box :active, disabled: !@current_user.admin?
+    = form.label :archived?
+    = form.check_box :archived?
```

Comment:
Did you know you can specify what the checked and unchecked values of a checkbox are?

```haml
= form.check_box :archived?, {}, 'archived', 'active'
```

Then that value will be in the `params` and you can use it directly.